io-package.json

```diff
@@ -303,6 +306,15 @@
                 "object": 1092
             }
         },
+        {
```

Comment:
As discussed in Telegram, we should not add that but only new one and the adapters update with a dep to controller 3.3.10+